src/aiida_wannier90_workflows/utils/projectors/upfdict.py

```diff
@@ -0,0 +1,122 @@
+from upf_tools import UPFDict
+from aiida_wannier90_workflows.utils.projectors.projectors import newProjector, newProjectors
```

Comment:
If this folder is moved into `dev/`, you can use

```suggestion
from projectors import newProjector, newProjectors
```